component/samples/ethereum/sample.bal

```diff
@@ -0,0 +1,224 @@
+//
+// Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Some files are in 2017 while others in 2018. Is that correct?

Reply:
No. It should be 2018. my mistake